webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -8804,7 +8804,7 @@ var userOverride = {
                         ? 'km-custom-widget-background-color'
                         : 'km-custom-widget-background-color-secondary';
 
-                if ( floatWhere !== 'mck-msg-right' && kmAttchMsg === 'km-attach-msg') {
+                if ( (floatWhere !== 'mck-msg-right' && kmAttchMsg === 'km-attach-msg') || msg.contentType == 8) {
```

Comment:
check how kmAttchMsg variable is getting set
we need to make the change there

Reply:
done